packages/debug-utils/index.js

```diff
@@ -530,13 +530,16 @@ var DebugUtils = {
     }
   },
 
-  formatInstruction: function (traceIndex, traceLength, instruction) {
+  formatCurrentInstruction: function (instruction) {
+    const pc = this.formatPC(instruction.pc);
+    const formattedInstruction = this.formatInstruction(instruction);
+    return (
+      truffleColors.mint("-> " + formattedInstruction + pc )
+    );
+  },
+
+  formatInstruction: function (instruction) {
```

Comment:
Note: This might be technically a breaking change to `debug-utils`... however that's fine, it's purely an internal package, so feel free to break it as much as you want. :)  It only matters when it comes time to do a release and you have to bump the version.